sales.html

```diff
@@ -5,9 +5,28 @@
     <title></title>
   </head>
   <body>
-    <section id="cookie-table">
+    <div class="">
+      <section id="cookie-table">
 
-    </section>
+      </section>
+    </div>
+
+    <div class="form">
```

Comment:
Something like "form-holder" or "form-wrapper" would probably be a better class name. In general, don't name your classes after html elements, especially if you're putting those classes on a different kind of element.